### DISCUSSION TITLE
Spectrum resolution

### DISCUSSION THREAD

**Discussion by ok1vve** [dl9rdz/rdz_ttgo_sonde#236] [Q&A] [answered]
Hello,
that is great :-)
everything works as it should. I used a 2.2-inch display, so I got angry with the display layout for a while. Now everything is beautiful, except for the spectrum. I only have it about halfway across the screen. Is it possible to make the spectrum bigger? There is nothing on the spectrum screen except descriptions.
Thanks for the answer and especially for the absolutely great sw

**Comment by dl9rdz** [comment, marked as answer]
Please provide more details (which version of the software? which display)

WIth any not-very-old version, the spectrum should look like this on any 2.0" and 2.2" ILI9225 display:
![s2-spec](https://user-images.githubusercontent.com/6459112/139557521-55b1e719-de41-4b0f-996d-ed348b9b9857.JPG)

For the high-resolution ILI9341 (usually 2.8" but some 2.2" display exist as well) the space is not fully utilized yet, and it should look like that (currently) (better use of space will come):
![s4-spec](https://user-images.githubusercontent.com/6459112/139557575-1b0c2d52-6083-4f74-975e-07b0ad5d1640.JPG)

For the small high-resolution ILI9342 display (M5Stack) it is currently broken (the tiny OLED configuration is used, so it is *really* tiny).

**Comment by dl9rdz** [comment]
Ok, so now it should be slightly better on ILI934x displays as well

2.8" ILI9341:
![s4-spec](https://user-images.githubusercontent.com/6459112/139558895-d9e175f9-facd-46d1-807e-8b4c66f3640a.JPG)

M5Stack ILI9342:
![s5-spec](https://user-images.githubusercontent.com/6459112/139558903-8d8a3f53-19e8-4bf5-a5fb-1c5343a5a9d9.JPG)

**Comment by ok1vve** [comment]
I download last week the last master verision.
My Display is:
2.2" 240x320 TFT displej, ILI9341, SPI
The spectrum looks like the second one from the first answer.
![139557575-1b0c2d52-6083-4f74-975e-07b0ad5d1640](https://user-images.githubusercontent.com/56109695/139594111-dd2b8b11-cc3a-45d9-98ad-8aa4f809ed8a.JPG)

**Comment by dl9rdz** [comment]
Yes, the new update is not yet in the master version, only in the latest devel version.

Probably some time in the next weeks I will merge the recent devel updates into master.

**Reply by ok1vve** [reply]
Thank you

**Comment by ok1vve** [comment]
you are the best
i updated on last Dev version and all works fine.
thank you
